### ISSUE TITLE
Support for interfaces

### ISSUE THREAD

**Issue by mjgoeke** [Evertras/typescript-cs-poco#28]
How difficult would it be to support exporting typescript interfaces for cs interface types?
e.g.

```cs
public interface IMessage { }
public interface IEvent : IMessage { }

public class Event : IEvent
{
    ...
}
```
I think the resulting ts should look like

```typescript
export interface IMessage { }
export interface IEvent extends IMessage { }
export interface Event extends IEvent {
    ...
 }
```

currently the interfaces aren't detected and exported, resulting in incomplete ts (missing the IEvent interface)

```typescript
export interface Event extends IEvent {
    ...
}
```

**Comment by Evertras** [comment]
Added a `includeInterfaces` option in `1.9.0`.
